doc/api/buffer.markdown

```diff
@@ -31,9 +31,8 @@ encoding method.  Here are the different string encodings.
 * `'base64'` - Base64 string encoding.
 
 * `'binary'` - A way of encoding raw binary data into strings by using only
-  the first 8 bits of each character. This encoding method is deprecated and
-  should be avoided in favor of `Buffer` objects where possible. This encoding
-  will be removed in future versions of Node.js.
+  the first 8 bits of each character. This encoding is synonymous with
+  `latin-1` encoding and can be used interchangeably.
```

Comment:
I'd reword this a little so people won't think `buf.toString('latin-1')` is a thing.